inc/search.class.php

```diff
@@ -2406,7 +2409,8 @@ static function displayCriteria($request = []) {
             ]);
          }
          echo "<i class='far fa-minus-square' alt='-' title=\"".
-                  __s('Delete a rule')."\" onclick=\"$('#$rowid').remove();\"></i>&nbsp;";
+                  __s('Delete a rule')."\" onclick=\"$('#$rowid').remove();
+                  $('#searchcriteria ul li:first-child').addClass('headerRow').show();\"></i>&nbsp;";
```

Comment:
PR seems good for.
One point here and for the two following, i think we need to remove `onclick` attributes (i miss to do it) and write in a (existing?) jquery block with click events